lib/internal/Magento/Framework/App/Action/AbstractAction.php

```diff
@@ -7,28 +7,35 @@
  */
 namespace Magento\Framework\App\Action;
 
+use Magento\Framework\App\ActionInterface;
 use Magento\Framework\App\RequestInterface;
 use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\Result\RedirectFactory;
+use Magento\Framework\Controller\ResultFactory;
```

Comment:
Please avoid such changes in future pull requests, it needs to be a separate effort rather than noise in regular diffs.

Reply:
I'm sorry about that.

Reply:
Removed noise for you.